api/rpc/account/account.go

```diff
@@ -215,6 +216,7 @@ func (s *Server) GetUser(ctx context.Context, in *GetUserRequest) (*GetUserReply
 }
 
 // ListUsers implements account.ListUsers
+// nolint : dupl
```

Comment:
I'm curious if we should try adjusting the threshold size (minimum token sequence) to eliminate so many of these false positives (see [DuplThreshold](https://github.com/alecthomas/gometalinter/blob/master/config.go#L184).